corehq/apps/users/views/mobile/users.py

```diff
@@ -97,6 +99,7 @@
                          "ManageCommCareMobileWorkers-B.UseBulkUploadtocreatem"
                          "ultipleusersatonce")
 DEFAULT_USER_LIST_LIMIT = 10
+BAD_MOBILE_USERNAME_REGEX = re.compile("[@!#%&'*/=?^`{}|]")
```

Comment:
Some of these symbols are technically allowed by django, but I figured it's better to be restrictive than not. This should only prevent users from creating users with these symbols in the future, not effect current users with these symbols

Reply:
Seems fine. I'd be in favor of making it even stricter and using something like `[^A-Za-z0-9_.+-]` (if you can tolerate not being able to make any additional j$ users).

Reply:
:wave:  future j$ users

 https://github.com/dimagi/commcare-hq/pull/20868/commits/2f30ab986613aa39bc1fe8626e4a9974f8fd099f

Reply:
new regex is missing `0-9`